test/middleware/test-errors.js

```diff
@@ -0,0 +1,43 @@
+const { assert } = require('chai');
+const app = require('express')();
+const config = require('../../app/config');
+const log = require('bole')('middleware');
+const errors_middleware = require('../../app/middleware/errors')(app, config, log);
+
+
+describe('When a "state mismatch" error occurs', () => {
+  it('redirects the user to /login', () => {
+    let spy_session_destroyed = false;
+    let spy_cookie_cleared = 'NOT CLEARED';
+    let spy_redirected_to = 'NOT REDIRECTED';
+    let spy_next_called = false;
+
+    const err = Error('state mismatch, could not find [...]');
```

Comment:
Is this unfinished? Should there be a variable interpolation instead of `[...]`?

Reply:
The original error message is quite long and we don't really care in the context of testing, so I just cut the original message as the logic only checks if the error message starts with `state mismatch`.

Reply:
👍